.github/workflows/prepare-release.yml

```diff
@@ -103,7 +97,7 @@ jobs:
 
       - name: Create release branch
         env:
-          RELEASE_BRANCH: releases/${{env.RELEASE_VERSION}}
+          RELEASE_BRANCH: releases/main/${{env.RELEASE_VERSION}}
```

Comment:
I'm not following why "main" needs to be hardcoded here

Reply:
I'm using it as a prefix to differentiate between main releases and `beta`/`rc` releases, so I can use it to filter out which workflow to run:
- [branches: ['releases/beta/**']](https://github.com/Activiti/activiti-scripts/pull/57/files#diff-8a77abef0638071fb764696102df451647100463f608ec7fe71bacc21e346db2R5) for beta releases
- [branches: ['releases/main/**']](https://github.com/Activiti/activiti-scripts/pull/57/files#diff-9a2542d199a8fdbfaf22c30b862c4fb101c221b85d411dde63b2eaee55e8f747R5) for main releases.

Reply:
makes sense, thanks :+1: